Sources/Fullscreen/FullscreenGallery/Transition/FullscreenGalleryTransitionPresenterDelegate.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright (c) 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+/// The Fullscreen Gallery Transition is a round-trip, and the Transition Presenter Delegate will be used
+/// both when transitioning in, and when transitioning out.
+public protocol FullscreenGalleryTransitionPresenterDelegate: class {
```

Comment:
As @osanoj mentioned in another PR, we should start using `AnyObject` over `class` for protocols. As mentioned in [this Swift Evolution proposal](https://github.com/apple/swift-evolution/blob/master/proposals/0156-subclass-existentials.md#class-and-anyobject), `class` may be removed at some point.

No worries at this point at all, but it could save us some changes when/if it'll be removed :)